tests/resolvers/User/user.spec.ts

```diff
@@ -0,0 +1,69 @@
+import "dotenv/config";
+import { connect, disconnect } from "../../helpers/db";
+import { beforeEach, describe, it, expect, beforeAll, afterAll } from "vitest";
+import { User } from "../../../src/models";
+import type mongoose from "mongoose";
+import { createTestUserFunc } from "../../helpers/user";
+import type { TestUserType } from "../../helpers/user";
+let MONGOOSE_INSTANCE: typeof mongoose;
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+});
+
+afterAll(async () => {
+  await disconnect(MONGOOSE_INSTANCE);
+});
+
+describe("User Identifier Tests", () => {
+  // before(async () => {
+  //   // Connect to MongoDB before running the tests
+  //   await mongoose.connect("mongodb://localhost:27017/talawa-test-db", {
+  //     useNewUrlParser: true,
+  //     useUnifiedTopology: true,
+  //   });
+  // });
+
+  // after(async () => {
+  //   // Disconnect from MongoDB after running the tests
+  //   await mongoose.connection.close();
+  // });
+
```

Comment:
Remove commented code is not useful for future

Reply:
i have fixed both sir
if there are any further changes required , I'd be happy to do them.